### PR TITLE
frontend: new landing page

### LINKED ISSUE #146: frontend: implement new landing page
image

### PR DESCRIPTION
**Pull request by bwdmonkey**
:tickets: **Ticket(s)**: Closes #146 

---

## :construction_worker: Changes

Screen Shot 2019-03-29 at 8 09 44 PM

## :flashlight: Testing Instructions
```
make web
```
Does it look similar on your screen?

### CODE REVIEW

**Comment by bobheadxi** [comment]
I'm getting an error and blank page:

image

also, might want to remove the blue/white pinpoint logo - sorry @LynxHack

File: frontend/src/styles/utils/spacing.scss [hunk 1]
```diff
@@ -10,21 +10,23 @@ $spacing-l:   32px;
...
-  &-nav { 
+  &-nav {
     padding-bottom: $spacing-nav;
   }
 
   // left padding
   &-left, &-sides {
     &-m { padding-left: $spacing-m; }
-    &-xxxl { padding-left: $spacing-xxxl; }
+    &-300 { padding-left: $spacing-300; }
```
Comment (bobheadxi) [inline frontend/src/styles/utils/spacing.scss]: ... this is a way better naming scheme than `xxxl` 👍 🏅

File: frontend/src/styles/utils/buttons.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  height: 30px;
+  width: 150px;
+}
+
+.button-signup {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  background: #EA0808;
+  border: #EA0808;
```
Comment (bobheadxi) [inline frontend/src/styles/utils/buttons.scss]: can these colours be variables? i think we'll be seeing these a lot, and should the scheme change it'll be easier to update

File: frontend/src/styles/utils/buttons.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  border-radius: 20px;
+  color: white;
+  width: 300px;
+  height: 60px;
+  font-size: 24px;
+  &:hover {
+    text-decoration: underline;
```
Comment (bobheadxi) [inline frontend/src/styles/utils/buttons.scss]: can we make this a separate `decorations.scss` class, and `@extend` it here? I imagine we want consistent button interactions, whether it be underlines (fine for now) or in the future, maybe a highlight colour / animation etc

Comment (leesw98) [reply frontend/src/styles/utils/buttons.scss]: This I will do as a separate issue as I have some ideas about this. Openned #157

Comment (bobheadxi) [reply frontend/src/styles/utils/buttons.scss]: sounds good 👌

File: frontend/src/styles/base/colors.scss [hunk 4]
```diff
@@ -3,3 +3,4 @@
 /////////////////////////
 
 $shadow: rgba(0, 82, 174, .15);
+$pinpoint-primary: #EA0808;
```
Comment (bobheadxi) [inline frontend/src/styles/base/colors.scss]: nitpick - I think it's safe to just call this `primary`:

```suggestion
$primary: #EA0808;
```

File: frontend/src/styles/base/base.scss [hunk 5]
```diff
@@ -31,4 +33,5 @@ html {
 body {
   height: 100%;
   width: 100%;
+  background: linear-gradient(180deg, #F20909 -8.73%, #FFA800 87.82%);
```
Comment (bobheadxi) [inline frontend/src/styles/base/base.scss]: maybe pop this into `$background` as well, just for consistency's sake

**Review by bobheadxi — APPROVED** [review]
other than build failure looks good

small note though, it looks like there's a brief "flicker" between pages where the background resets to white and then back to gradient when I click "sign up" - is this avoidable?

**Comment by leesw98** [comment]
@bobheadxi I can't reproduce the issue. What browser/steps?

**Comment by bobheadxi** [comment]
@leesw98 chrome. click "sign up for free"

**Pull request merged**